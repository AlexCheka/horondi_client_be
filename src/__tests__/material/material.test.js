/* eslint-disable no-undef */
const { gql } = require('apollo-boost');
const materialService = require('../../modules/material/material.service')
const materialResolver = require('../../modules/material/material.resolver')
const client = require('../../utils/apolloClient');

require('dotenv').config();


describe('querries', () => {
    test('should recive all materials like in service and resolver', async () => {
        const res = await client.query({
            query: gql`
                query {       
                    getAllMaterials {
                        _id
                        name{
                            lang
                            value
                        }
                        description{
                            lang
                            value
                        }
                        colors{
                            code
                            name{
                                lang
                                value
                            }
                            images{
                                large
                                medium
                                small
                                thumbnail
                            }
                            available
                        }
                        available
                        additionalPrice
                    }
                }
            `
        }).then(res => res)
        .catch(err => err)

        expect(materialService.getAllMaterials()).resolves.toBe(res);
        expect(materialResolver.materialQuery.getAllMaterials()).resolves.toBe(res);
        expect(res.data.getAllMaterials).toMatchSnapshot()
    })

    test('should recive material by id like in service and resolver', async () => {
      const res = await client.query({
          query: gql`
              query {       
                getMaterialById(id: "1d2bba5d0b80938327ac9012"){
                    _id
                        name{
                            lang
                            value
                        }
                        description{
                            lang
                            value
                        }
                        colors{
                            code
                            name{
                                lang
                                value
                            }
                            images{
                                large
                                medium
                                small
                                thumbnail
                            }
                            available
                        }
                        available
                        additionalPrice
                }
              }
          `
      }).then(res => res)
      .catch(err => err)

      expect(materialService.getMaterialById("1d2bba5d0b80938327ac9012")).resolves.toBe(res);
      expect(materialResolver.materialQuery.getMaterialById('',{id:"1d2bba5d0b80938327ac9012"})).resolves.toBe(res);
      expect(res.data.getMaterialById).toMatchSnapshot()
  })
})