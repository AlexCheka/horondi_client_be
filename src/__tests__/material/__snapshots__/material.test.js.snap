// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`querries should recive all materials like in service and resolver 1`] = `
Array [
  Object {
    "_id": "56ade69dd46eafc5968e5390",
    "additionalPrice": 0,
    "available": true,
    "colors": Array [
      Object {
        "available": true,
        "code": 200,
        "images": Object {
          "large": "large-light-brown.jpg",
          "medium": "medium-light-brown.jpg",
          "small": "small-light-brown.jpg",
          "thumbnail": "thumbnail-light-brown.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Світло-коричневий",
          },
          Object {
            "lang": "en",
            "value": "Light-brown",
          },
        ],
      },
      Object {
        "available": true,
        "code": 201,
        "images": Object {
          "large": "large-rosy-brown.jpg",
          "medium": "medium-rosy-brown.jpg",
          "small": "small-rosy-brown.jpg",
          "thumbnail": "thumbnail-rosy-brown.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Рожево-коричневий",
          },
          Object {
            "lang": "en",
            "value": "Rosy-brown",
          },
        ],
      },
      Object {
        "available": true,
        "code": 202,
        "images": Object {
          "large": "large-saddle-brown.jpg",
          "medium": "medium-saddle-brown.jpg",
          "small": "small-saddle-brown.jpg",
          "thumbnail": "thumbnail-saddle-brown.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Насичений коричневий",
          },
          Object {
            "lang": "en",
            "value": "Saddle-brown",
          },
        ],
      },
      Object {
        "available": true,
        "code": 203,
        "images": Object {
          "large": "large-dark-brown.jpg",
          "medium": "medium-dark-brown.jpg",
          "small": "small-dark-brown.jpg",
          "thumbnail": "thumbnail-dark-brown.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Темно-коричневий",
          },
          Object {
            "lang": "en",
            "value": "Dark-brown",
          },
        ],
      },
      Object {
        "available": true,
        "code": 204,
        "images": Object {
          "large": "large-light-pink.jpg",
          "medium": "medium-light-pink.jpg",
          "small": "small-light-pink.jpg",
          "thumbnail": "thumbnail-light-pink.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Світло-рожевий",
          },
          Object {
            "lang": "en",
            "value": "Light-pink",
          },
        ],
      },
      Object {
        "available": true,
        "code": 205,
        "images": Object {
          "large": "large-violet-red.jpg",
          "medium": "medium-violet-red.jpg",
          "small": "small-violet-red.jpg",
          "thumbnail": "thumbnail-violet-red.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Вишневий",
          },
          Object {
            "lang": "en",
            "value": "Violet-red",
          },
        ],
      },
      Object {
        "available": true,
        "code": 206,
        "images": Object {
          "large": "large-golden.jpg",
          "medium": "medium-golden.jpg",
          "small": "small-golden.jpg",
          "thumbnail": "thumbnail-golden.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Золотий",
          },
          Object {
            "lang": "en",
            "value": "Golden",
          },
        ],
      },
      Object {
        "available": true,
        "code": 207,
        "images": Object {
          "large": "large-dark-olive.jpg",
          "medium": "medium-dark-olive.jpg",
          "small": "small-dark-olive.jpg",
          "thumbnail": "thumbnail-dark-olive.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Темно-оливковий",
          },
          Object {
            "lang": "en",
            "value": "Dark-olive",
          },
        ],
      },
      Object {
        "available": true,
        "code": 208,
        "images": Object {
          "large": "large-light-steel.jpg",
          "medium": "medium-light-steel.jpg",
          "small": "small-light-steel.jpg",
          "thumbnail": "thumbnail-light-steel.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Світло-сталевий",
          },
          Object {
            "lang": "en",
            "value": "Light-steel",
          },
        ],
      },
      Object {
        "available": true,
        "code": 209,
        "images": Object {
          "large": "large-steel-blue.jpg",
          "medium": "medium-steel-blue.jpg",
          "small": "small-steel-blue.jpg",
          "thumbnail": "thumbnail-steel-blue.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Сталево-блакитний",
          },
          Object {
            "lang": "en",
            "value": "Steel-blue",
          },
        ],
      },
      Object {
        "available": true,
        "code": 210,
        "images": Object {
          "large": "large-dark-blue.jpg",
          "medium": "medium-dark-blue.jpg",
          "small": "small-dark-blue.jpg",
          "thumbnail": "thumbnail-dark-blue.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Темно-синій",
          },
          Object {
            "lang": "en",
            "value": "Dark-blue",
          },
        ],
      },
      Object {
        "available": true,
        "code": 211,
        "images": Object {
          "large": "large-light-gray.jpg",
          "medium": "medium-light-gray.jpg",
          "small": "small-light-gray.jpg",
          "thumbnail": "thumbnail-light-gray.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Світло-сірий",
          },
          Object {
            "lang": "en",
            "value": "Light-gray",
          },
        ],
      },
      Object {
        "available": true,
        "code": 212,
        "images": Object {
          "large": "large-silver.jpg",
          "medium": "medium-silver.jpg",
          "small": "small-silver.jpg",
          "thumbnail": "thumbnail-silver.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Срібний",
          },
          Object {
            "lang": "en",
            "value": "Silver",
          },
        ],
      },
      Object {
        "available": true,
        "code": 213,
        "images": Object {
          "large": "large-slate-gray.jpg",
          "medium": "medium-slate-gray.jpg",
          "small": "small-slate-gray.jpg",
          "thumbnail": "thumbnail-slate-gray.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Сизий",
          },
          Object {
            "lang": "en",
            "value": "Slate-gray",
          },
        ],
      },
      Object {
        "available": true,
        "code": 214,
        "images": Object {
          "large": "large-dim-gray.jpg",
          "medium": "medium-dim-gray.jpg",
          "small": "small-dim-gray.jpg",
          "thumbnail": "thumbnail-dim-gray.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Брудно-сірий",
          },
          Object {
            "lang": "en",
            "value": "Dim-gray",
          },
        ],
      },
      Object {
        "available": true,
        "code": 215,
        "images": Object {
          "large": "large-dark-gray.jpg",
          "medium": "medium-dark-gray.jpg",
          "small": "small-dark-gray.jpg",
          "thumbnail": "thumbnail-dark-gray.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Темно-сірий",
          },
          Object {
            "lang": "en",
            "value": "Dark-gray",
          },
        ],
      },
    ],
    "description": Array [
      Object {
        "lang": "uk",
        "value": "Мальмо (100% поліестер) - для рюкзаків, сумок, бананок (всі моделі). Прошита додатковим шаром спеціального матеріалу, який зміцнює та захищає від води.",
      },
      Object {
        "lang": "en",
        "value": "Malmo (100% polyester) - for backpacks, bags, fanny-packs (all models). Layered with special material for durability and water-resistance.",
      },
    ],
    "name": Array [
      Object {
        "lang": "uk",
        "value": "Мальмо",
      },
      Object {
        "lang": "en",
        "value": "Malmo",
      },
    ],
  },
  Object {
    "_id": "1d2bba5d0b80938327ac9012",
    "additionalPrice": 0,
    "available": true,
    "colors": Array [
      Object {
        "available": true,
        "code": 300,
        "images": Object {
          "large": "large-linen.jpg",
          "medium": "medium-linen.jpg",
          "small": "small-linen.jpg",
          "thumbnail": "thumbnail-linen.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Лляний",
          },
          Object {
            "lang": "en",
            "value": "Linen",
          },
        ],
      },
      Object {
        "available": true,
        "code": 301,
        "images": Object {
          "large": "large-wheat.jpg",
          "medium": "medium-wheat.jpg",
          "small": "small-wheat.jpg",
          "thumbnail": "thumbnail-wheat.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Пшеничний",
          },
          Object {
            "lang": "en",
            "value": "Wheat",
          },
        ],
      },
      Object {
        "available": true,
        "code": 302,
        "images": Object {
          "large": "large-light-salmon.jpg",
          "medium": "medium-light-salmon.jpg",
          "small": "small-light-salmon.jpg",
          "thumbnail": "thumbnail-light-salmon.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Світло-лососевий",
          },
          Object {
            "lang": "en",
            "value": "Light-salmon",
          },
        ],
      },
      Object {
        "available": true,
        "code": 303,
        "images": Object {
          "large": "large-salmon.jpg",
          "medium": "medium-salmon.jpg",
          "small": "small-salmon.jpg",
          "thumbnail": "thumbnail-salmon.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Лососевий",
          },
          Object {
            "lang": "en",
            "value": "Salmon",
          },
        ],
      },
      Object {
        "available": true,
        "code": 304,
        "images": Object {
          "large": "large-dark-salmon.jpg",
          "medium": "medium-dark-salmon.jpg",
          "small": "small-dark-salmon.jpg",
          "thumbnail": "thumbnail-dark-salmon.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Темно-лососевий",
          },
          Object {
            "lang": "en",
            "value": "Dark-salmon",
          },
        ],
      },
      Object {
        "available": true,
        "code": 305,
        "images": Object {
          "large": "large-saddle-brown.jpg",
          "medium": "medium-saddle-brown.jpg",
          "small": "small-saddle-brown.jpg",
          "thumbnail": "thumbnail-saddle-brown.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Насичений коричневий",
          },
          Object {
            "lang": "en",
            "value": "Saddle-brown",
          },
        ],
      },
      Object {
        "available": true,
        "code": 306,
        "images": Object {
          "large": "large-light-slate-gray.jpg",
          "medium": "medium-light-slate-gray.jpg",
          "small": "small-light-slate-gray.jpg",
          "thumbnail": "thumbnail-light-slate-gray.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Світло-сизий",
          },
          Object {
            "lang": "en",
            "value": "Light-slate-gray",
          },
        ],
      },
      Object {
        "available": true,
        "code": 307,
        "images": Object {
          "large": "large-whitesmoke.jpg",
          "medium": "medium-whitesmoke.jpg",
          "small": "small-whitesmoke.jpg",
          "thumbnail": "thumbnail-whitesmoke.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Димчатий",
          },
          Object {
            "lang": "en",
            "value": "Whitesmoke",
          },
        ],
      },
      Object {
        "available": true,
        "code": 308,
        "images": Object {
          "large": "large-maroon.jpg",
          "medium": "medium-maroon.jpg",
          "small": "small-maroon.jpg",
          "thumbnail": "thumbnail-maroon.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Бордовий",
          },
          Object {
            "lang": "en",
            "value": "Maroon",
          },
        ],
      },
      Object {
        "available": true,
        "code": 309,
        "images": Object {
          "large": "large-royal-blue.jpg",
          "medium": "medium-royal-blue.jpg",
          "small": "small-royal-blue.jpg",
          "thumbnail": "thumbnail-royal-blue.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Королівський синій",
          },
          Object {
            "lang": "en",
            "value": "Royal-blue",
          },
        ],
      },
      Object {
        "available": true,
        "code": 310,
        "images": Object {
          "large": "large-coral.jpg",
          "medium": "medium-coral.jpg",
          "small": "small-coral.jpg",
          "thumbnail": "thumbnail-coral.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Кораловий",
          },
          Object {
            "lang": "en",
            "value": "Coral",
          },
        ],
      },
      Object {
        "available": true,
        "code": 311,
        "images": Object {
          "large": "large-orange.jpg",
          "medium": "medium-orange.jpg",
          "small": "small-orange.jpg",
          "thumbnail": "thumbnail-orange.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Оранжевий",
          },
          Object {
            "lang": "en",
            "value": "Orange",
          },
        ],
      },
      Object {
        "available": true,
        "code": 312,
        "images": Object {
          "large": "large-gold.jpg",
          "medium": "medium-gold.jpg",
          "small": "small-gold.jpg",
          "thumbnail": "thumbnail-gold.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Золотий",
          },
          Object {
            "lang": "en",
            "value": "Gold",
          },
        ],
      },
      Object {
        "available": true,
        "code": 313,
        "images": Object {
          "large": "large-khaki.jpg",
          "medium": "medium-khaki.jpg",
          "small": "small-khaki.jpg",
          "thumbnail": "thumbnail-khaki.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Хакі",
          },
          Object {
            "lang": "en",
            "value": "Khaki",
          },
        ],
      },
      Object {
        "available": true,
        "code": 314,
        "images": Object {
          "large": "large-dark-khaki.jpg",
          "medium": "medium-dark-khaki.jpg",
          "small": "small-dark-khaki.jpg",
          "thumbnail": "thumbnail-dark-khaki.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Темний хакі",
          },
          Object {
            "lang": "en",
            "value": "Dark-khaki",
          },
        ],
      },
      Object {
        "available": true,
        "code": 315,
        "images": Object {
          "large": "large-pale-violet.jpg",
          "medium": "medium-pale-violet.jpg",
          "small": "small-pale-violet.jpg",
          "thumbnail": "thumbnail-pale-violet.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Блідо-фіолетовий",
          },
          Object {
            "lang": "en",
            "value": "Pale-violet",
          },
        ],
      },
      Object {
        "available": true,
        "code": 316,
        "images": Object {
          "large": "large-deep-sky-blue.jpg",
          "medium": "medium-deep-sky-blue.jpg",
          "small": "small-deep-sky-blue.jpg",
          "thumbnail": "thumbnail-deep-sky-blue.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Глибокий небесно-синій",
          },
          Object {
            "lang": "en",
            "value": "Deep-sky-blue",
          },
        ],
      },
      Object {
        "available": true,
        "code": 317,
        "images": Object {
          "large": "large-midnight-blue.jpg",
          "medium": "medium-midnight-blue.jpg",
          "small": "small-midnight-blue.jpg",
          "thumbnail": "thumbnail-midnight-blue.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Темно-синій",
          },
          Object {
            "lang": "en",
            "value": "Midnight-blue",
          },
        ],
      },
      Object {
        "available": true,
        "code": 318,
        "images": Object {
          "large": "large-dark-red.jpg",
          "medium": "medium-dark-red.jpg",
          "small": "small-dark-red.jpg",
          "thumbnail": "thumbnail-dark-red.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Темно-червоний",
          },
          Object {
            "lang": "en",
            "value": "Dark-red",
          },
        ],
      },
    ],
    "description": Array [
      Object {
        "lang": "uk",
        "value": "Бонд (100% поліестер) - для рюкзаків моделі Rolltop. Прошита додатковим шаром спеціального матеріалу, який зміцнює та захищає від води.",
      },
      Object {
        "lang": "en",
        "value": "Bond (100% polyester) - for backpacks of Rolltop model. Layered with special material for durability and water-resistance.",
      },
    ],
    "name": Array [
      Object {
        "lang": "uk",
        "value": "Бонд",
      },
      Object {
        "lang": "en",
        "value": "Bond",
      },
    ],
  },
  Object {
    "_id": "6fdbc0fd73001f63c83226f5",
    "additionalPrice": 0,
    "available": true,
    "colors": Array [
      Object {
        "available": true,
        "code": 1,
        "images": Object {
          "large": "large-black.jpg",
          "medium": "medium-black.jpg",
          "small": "small-black.jpg",
          "thumbnail": "thumbnail-black.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Чорний",
          },
          Object {
            "lang": "en",
            "value": "Black",
          },
        ],
      },
      Object {
        "available": true,
        "code": 2,
        "images": Object {
          "large": "large-brown.jpg",
          "medium": "medium-brown.jpg",
          "small": "small-brown.jpg",
          "thumbnail": "thumbnail-brown.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Коричневий",
          },
          Object {
            "lang": "en",
            "value": "Brown",
          },
        ],
      },
      Object {
        "available": true,
        "code": 3,
        "images": Object {
          "large": "large-gray.jpg",
          "medium": "medium-gray.jpg",
          "small": "small-gray.jpg",
          "thumbnail": "thumbnail-gray.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Сірий",
          },
          Object {
            "lang": "en",
            "value": "Gray",
          },
        ],
      },
      Object {
        "available": true,
        "code": 4,
        "images": Object {
          "large": "large-pink.jpg",
          "medium": "medium-pink.jpg",
          "small": "small-pink.jpg",
          "thumbnail": "thumbnail-pink.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Рожевий",
          },
          Object {
            "lang": "en",
            "value": "Pink",
          },
        ],
      },
    ],
    "description": Array [
      Object {
        "lang": "uk",
        "value": "Шкірзамінник",
      },
      Object {
        "lang": "en",
        "value": "Faux leather",
      },
    ],
    "name": Array [
      Object {
        "lang": "uk",
        "value": "Шкірзамінник",
      },
      Object {
        "lang": "en",
        "value": "Faux leather",
      },
    ],
  },
  Object {
    "_id": "dadba32060da96e40847166d",
    "additionalPrice": 0,
    "available": true,
    "colors": Array [
      Object {
        "available": true,
        "code": 1,
        "images": Object {
          "large": "large-black.jpg",
          "medium": "medium-black.jpg",
          "small": "small-black.jpg",
          "thumbnail": "thumbnail-black.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Чорний",
          },
          Object {
            "lang": "en",
            "value": "Black",
          },
        ],
      },
    ],
    "description": Array [
      Object {
        "lang": "uk",
        "value": "Тканина Кордура",
      },
      Object {
        "lang": "en",
        "value": "Cordura fabric",
      },
    ],
    "name": Array [
      Object {
        "lang": "uk",
        "value": "Тканина Кордура",
      },
      Object {
        "lang": "en",
        "value": "Cordura fabric",
      },
    ],
  },
  Object {
    "_id": "88cb31bf8e5ea5af3014e342",
    "additionalPrice": 0,
    "available": true,
    "colors": Array [
      Object {
        "available": true,
        "code": 1,
        "images": Object {
          "large": "large-black.jpg",
          "medium": "medium-black.jpg",
          "small": "small-black.jpg",
          "thumbnail": "thumbnail-black.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Чорний",
          },
          Object {
            "lang": "en",
            "value": "Black",
          },
        ],
      },
      Object {
        "available": true,
        "code": 2,
        "images": Object {
          "large": "large-brown.jpg",
          "medium": "medium-brown.jpg",
          "small": "small-brown.jpg",
          "thumbnail": "thumbnail-brown.jpg",
        },
        "name": Array [
          Object {
            "lang": "uk",
            "value": "Коричневий",
          },
          Object {
            "lang": "en",
            "value": "Brown",
          },
        ],
      },
    ],
    "description": Array [
      Object {
        "lang": "uk",
        "value": "Натуральна шкіра",
      },
      Object {
        "lang": "en",
        "value": "Genuine leather",
      },
    ],
    "name": Array [
      Object {
        "lang": "uk",
        "value": "Натуральна шкіра",
      },
      Object {
        "lang": "en",
        "value": "Genuine leather",
      },
    ],
  },
]
`;

exports[`querries should recive material by id like in service and resolver 1`] = `
Object {
  "_id": "1d2bba5d0b80938327ac9012",
  "additionalPrice": 0,
  "available": true,
  "colors": Array [
    Object {
      "available": true,
      "code": 300,
      "images": Object {
        "large": "large-linen.jpg",
        "medium": "medium-linen.jpg",
        "small": "small-linen.jpg",
        "thumbnail": "thumbnail-linen.jpg",
      },
      "name": Array [
        Object {
          "lang": "uk",
          "value": "Лляний",
        },
        Object {
          "lang": "en",
          "value": "Linen",
        },
      ],
    },
    Object {
      "available": true,
      "code": 301,
      "images": Object {
        "large": "large-wheat.jpg",
        "medium": "medium-wheat.jpg",
        "small": "small-wheat.jpg",
        "thumbnail": "thumbnail-wheat.jpg",
      },
      "name": Array [
        Object {
          "lang": "uk",
          "value": "Пшеничний",
        },
        Object {
          "lang": "en",
          "value": "Wheat",
        },
      ],
    },
    Object {
      "available": true,
      "code": 302,
      "images": Object {
        "large": "large-light-salmon.jpg",
        "medium": "medium-light-salmon.jpg",
        "small": "small-light-salmon.jpg",
        "thumbnail": "thumbnail-light-salmon.jpg",
      },
      "name": Array [
        Object {
          "lang": "uk",
          "value": "Світло-лососевий",
        },
        Object {
          "lang": "en",
          "value": "Light-salmon",
        },
      ],
    },
    Object {
      "available": true,
      "code": 303,
      "images": Object {
        "large": "large-salmon.jpg",
        "medium": "medium-salmon.jpg",
        "small": "small-salmon.jpg",
        "thumbnail": "thumbnail-salmon.jpg",
      },
      "name": Array [
        Object {
          "lang": "uk",
          "value": "Лососевий",
        },
        Object {
          "lang": "en",
          "value": "Salmon",
        },
      ],
    },
    Object {
      "available": true,
      "code": 304,
      "images": Object {
        "large": "large-dark-salmon.jpg",
        "medium": "medium-dark-salmon.jpg",
        "small": "small-dark-salmon.jpg",
        "thumbnail": "thumbnail-dark-salmon.jpg",
      },
      "name": Array [
        Object {
          "lang": "uk",
          "value": "Темно-лососевий",
        },
        Object {
          "lang": "en",
          "value": "Dark-salmon",
        },
      ],
    },
    Object {
      "available": true,
      "code": 305,
      "images": Object {
        "large": "large-saddle-brown.jpg",
        "medium": "medium-saddle-brown.jpg",
        "small": "small-saddle-brown.jpg",
        "thumbnail": "thumbnail-saddle-brown.jpg",
      },
      "name": Array [
        Object {
          "lang": "uk",
          "value": "Насичений коричневий",
        },
        Object {
          "lang": "en",
          "value": "Saddle-brown",
        },
      ],
    },
    Object {
      "available": true,
      "code": 306,
      "images": Object {
        "large": "large-light-slate-gray.jpg",
        "medium": "medium-light-slate-gray.jpg",
        "small": "small-light-slate-gray.jpg",
        "thumbnail": "thumbnail-light-slate-gray.jpg",
      },
      "name": Array [
        Object {
          "lang": "uk",
          "value": "Світло-сизий",
        },
        Object {
          "lang": "en",
          "value": "Light-slate-gray",
        },
      ],
    },
    Object {
      "available": true,
      "code": 307,
      "images": Object {
        "large": "large-whitesmoke.jpg",
        "medium": "medium-whitesmoke.jpg",
        "small": "small-whitesmoke.jpg",
        "thumbnail": "thumbnail-whitesmoke.jpg",
      },
      "name": Array [
        Object {
          "lang": "uk",
          "value": "Димчатий",
        },
        Object {
          "lang": "en",
          "value": "Whitesmoke",
        },
      ],
    },
    Object {
      "available": true,
      "code": 308,
      "images": Object {
        "large": "large-maroon.jpg",
        "medium": "medium-maroon.jpg",
        "small": "small-maroon.jpg",
        "thumbnail": "thumbnail-maroon.jpg",
      },
      "name": Array [
        Object {
          "lang": "uk",
          "value": "Бордовий",
        },
        Object {
          "lang": "en",
          "value": "Maroon",
        },
      ],
    },
    Object {
      "available": true,
      "code": 309,
      "images": Object {
        "large": "large-royal-blue.jpg",
        "medium": "medium-royal-blue.jpg",
        "small": "small-royal-blue.jpg",
        "thumbnail": "thumbnail-royal-blue.jpg",
      },
      "name": Array [
        Object {
          "lang": "uk",
          "value": "Королівський синій",
        },
        Object {
          "lang": "en",
          "value": "Royal-blue",
        },
      ],
    },
    Object {
      "available": true,
      "code": 310,
      "images": Object {
        "large": "large-coral.jpg",
        "medium": "medium-coral.jpg",
        "small": "small-coral.jpg",
        "thumbnail": "thumbnail-coral.jpg",
      },
      "name": Array [
        Object {
          "lang": "uk",
          "value": "Кораловий",
        },
        Object {
          "lang": "en",
          "value": "Coral",
        },
      ],
    },
    Object {
      "available": true,
      "code": 311,
      "images": Object {
        "large": "large-orange.jpg",
        "medium": "medium-orange.jpg",
        "small": "small-orange.jpg",
        "thumbnail": "thumbnail-orange.jpg",
      },
      "name": Array [
        Object {
          "lang": "uk",
          "value": "Оранжевий",
        },
        Object {
          "lang": "en",
          "value": "Orange",
        },
      ],
    },
    Object {
      "available": true,
      "code": 312,
      "images": Object {
        "large": "large-gold.jpg",
        "medium": "medium-gold.jpg",
        "small": "small-gold.jpg",
        "thumbnail": "thumbnail-gold.jpg",
      },
      "name": Array [
        Object {
          "lang": "uk",
          "value": "Золотий",
        },
        Object {
          "lang": "en",
          "value": "Gold",
        },
      ],
    },
    Object {
      "available": true,
      "code": 313,
      "images": Object {
        "large": "large-khaki.jpg",
        "medium": "medium-khaki.jpg",
        "small": "small-khaki.jpg",
        "thumbnail": "thumbnail-khaki.jpg",
      },
      "name": Array [
        Object {
          "lang": "uk",
          "value": "Хакі",
        },
        Object {
          "lang": "en",
          "value": "Khaki",
        },
      ],
    },
    Object {
      "available": true,
      "code": 314,
      "images": Object {
        "large": "large-dark-khaki.jpg",
        "medium": "medium-dark-khaki.jpg",
        "small": "small-dark-khaki.jpg",
        "thumbnail": "thumbnail-dark-khaki.jpg",
      },
      "name": Array [
        Object {
          "lang": "uk",
          "value": "Темний хакі",
        },
        Object {
          "lang": "en",
          "value": "Dark-khaki",
        },
      ],
    },
    Object {
      "available": true,
      "code": 315,
      "images": Object {
        "large": "large-pale-violet.jpg",
        "medium": "medium-pale-violet.jpg",
        "small": "small-pale-violet.jpg",
        "thumbnail": "thumbnail-pale-violet.jpg",
      },
      "name": Array [
        Object {
          "lang": "uk",
          "value": "Блідо-фіолетовий",
        },
        Object {
          "lang": "en",
          "value": "Pale-violet",
        },
      ],
    },
    Object {
      "available": true,
      "code": 316,
      "images": Object {
        "large": "large-deep-sky-blue.jpg",
        "medium": "medium-deep-sky-blue.jpg",
        "small": "small-deep-sky-blue.jpg",
        "thumbnail": "thumbnail-deep-sky-blue.jpg",
      },
      "name": Array [
        Object {
          "lang": "uk",
          "value": "Глибокий небесно-синій",
        },
        Object {
          "lang": "en",
          "value": "Deep-sky-blue",
        },
      ],
    },
    Object {
      "available": true,
      "code": 317,
      "images": Object {
        "large": "large-midnight-blue.jpg",
        "medium": "medium-midnight-blue.jpg",
        "small": "small-midnight-blue.jpg",
        "thumbnail": "thumbnail-midnight-blue.jpg",
      },
      "name": Array [
        Object {
          "lang": "uk",
          "value": "Темно-синій",
        },
        Object {
          "lang": "en",
          "value": "Midnight-blue",
        },
      ],
    },
    Object {
      "available": true,
      "code": 318,
      "images": Object {
        "large": "large-dark-red.jpg",
        "medium": "medium-dark-red.jpg",
        "small": "small-dark-red.jpg",
        "thumbnail": "thumbnail-dark-red.jpg",
      },
      "name": Array [
        Object {
          "lang": "uk",
          "value": "Темно-червоний",
        },
        Object {
          "lang": "en",
          "value": "Dark-red",
        },
      ],
    },
  ],
  "description": Array [
    Object {
      "lang": "uk",
      "value": "Бонд (100% поліестер) - для рюкзаків моделі Rolltop. Прошита додатковим шаром спеціального матеріалу, який зміцнює та захищає від води.",
    },
    Object {
      "lang": "en",
      "value": "Bond (100% polyester) - for backpacks of Rolltop model. Layered with special material for durability and water-resistance.",
    },
  ],
  "name": Array [
    Object {
      "lang": "uk",
      "value": "Бонд",
    },
    Object {
      "lang": "en",
      "value": "Bond",
    },
  ],
}
`;
